recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - osx-dist.patch       # [osx]
+    - win-find_exe.patch   # [win]
+
+build:
+  no_link: 
+    - bin/python
+  number: 0
+
+requirements:
+  build:
+    - bzip2     # [unix]
+    - zlib      # [unix]
```

Comment:
Looks like this may be possible but is not easy.  The PCbuild/readme.txt file of the Python source has this to say on the topic.

```
Getting External Sources
------------------------

The last category of sub-projects listed above wrap external projects
Python doesn't control, and as such a little more work is required in
order to download the relevant source files for each project before they
can be built.  However, a simple script is provided to make this as
painless as possible, called "get_externals.bat" and located in this
directory.  This script extracts all the external sub-projects from
    http://svn.python.org/projects/external
via Subversion (so you'll need svn.exe on your PATH) and places them
in ..\externals (relative to this directory).

It is also possible to download sources from each project's homepage,
though you may have to change folder names or pass the names to MSBuild
as the values of certain properties in order for the build solution to
find them.  This is an advanced topic and not necessarily fully
supported.
```